extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -243,29 +279,59 @@ export class AmpAnalytics extends AMP.BaseElement {
     });
 
     // For consistentcy with amp-pixel we also expand any url replacements.
-    urlReplacementsFor(this.getWin()).expand(request).then(request => {
-      // TODO(btownsend, #1061): Add support for sendBeacon.
-      if (host && request) {
-        this.sendRequest_('https://' + host + request);
-      }
+    urlReplacementsFor(this.getWin()).expand(data).then(data => {
+      this.sendRequest_({host: request.host, path: request.path, data: data});
     });
   }
 
   /**
-   * Sends a request via GET method.
-   *
-   * @param {!string} request The request to be sent over wire.
+   * @param {!RequestData}
    * @private
    */
   sendRequest_(request) {
+    if (!request.host) {
+      log.warn(this.getName_(),
+          'Failed to send request. No host specified in config.');
+      return;
+    }
+    const transport = this.config_['transport'] || 'image';
```

Comment:
Good point. I think a polyfill would be best. Is the browser coverage for that (http://caniuse.com/#search=cors) acceptable?